builder/dockerfile/containerbackend.go

```diff
@@ -0,0 +1,156 @@
+package dockerfile
```

Comment:
Should move implementation in this file out of `/builder`. To daemon or a subpkg of daemon.